src/Middleware/SpaServices.Extensions/src/AngularCli/AngularCliBuilder.cs

```diff
@@ -21,25 +21,26 @@ public class AngularCliBuilder : ISpaPrerendererBuilder
     {
         private static TimeSpan RegexMatchTimeout = TimeSpan.FromSeconds(5); // This is a development-time only feature, so a very long timeout is fine
 
-        private readonly string _npmScriptName;
+        private readonly string _scriptName;
 
         /// <summary>
         /// Constructs an instance of <see cref="AngularCliBuilder"/>.
         /// </summary>
-        /// <param name="npmScript">The name of the script in your package.json file that builds the server-side bundle for your Angular application.</param>
-        public AngularCliBuilder(string npmScript)
+        /// <param name="scriptName">The name of the script in your package.json file that builds the server-side bundle for your Angular application.</param>
+        public AngularCliBuilder(string scriptName)
```

Comment:
FYI this is a [small] breaking change. (Which is allowed in a major version update, but scrutinized closely.)

Reply:
I thought it made more sense since npmScript implies the use of npm, but I can change the name back for backwards compatability

Reply:
I agree the new name makes sense, I was just pointing out that it's potentially problematic. Always a trade-off 😄

Reply:
This change has been reverted